deploy/platform/platform_docker_build.sh

```diff
@@ -7,9 +7,10 @@
 # Build script from within a docker image
 #
 export HOME=/tmp/home
+srcdir=$(realpath $(dirname ${0})/../..)
```

Comment:
could be an option passed by build.sh
if [ -z  $srcdir ]
then srcdir=/source
fi